### ISSUE TITLE
Goodbye, world! Last Words Spoken By Criminals Before Execution

### ISSUE THREAD

**Issue by SiruiZhu** [jsoma/data-studio-projects#262]
# Pitch

## Summary

There is a dataset about death row information, and it has last statement for people before getting death execution. Because this is a sad dataset.  I want to do another text analysis before I give up on it  with my sorrowful heart. >...<

## Details
I'm simply interested in:

- What the most frequent word been mentioned in last statement?  I expected more like regretful words. But still want a try to see it. 
- I read through some of the sentence, it seems religion words been mentioned a lot(?) Is there a trend for religion words mentioned times over the years? Is the religion words related to race? 

**Possible headline(s):**
Goodbye, world!
Last Words Spoken By Criminals Before Execution

**Data set(s):** 
[Texas Department of Criminal Justice](http://www.tdcj.state.tx.us/death_row/dr_executed_offenders.html)

**Code repository:** 
https://github.com/SiruiZhu/Data-Studio/tree/master/code/05-death%20statement
	
**Possible problems/fears/questions:** 
Don't think I'll come up with nice visuals, but want to try another text analysis, just get familiar with it, also collect this dataset for other stories later, like sentimental analysis etc. 

## Work so far
Just scrap all the statement. I want something look like this:

###  Inspiration:
![sample1](https://user-images.githubusercontent.com/39416241/43752713-4c4870c4-99d0-11e8-8154-66102b9ce3e7.png)
![sample2](https://user-images.githubusercontent.com/39416241/43752715-4c58e9ea-99d0-11e8-8ada-8f0e80d1aec0.png)

## Checklist

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [ ] My issue explains what I'd like to explore in the data set
- [x] My issue includes images - either inspiration or what I've done so far

**Comment by SiruiZhu** [comment]
# Update 1
The word frequency of the most spoken words in last statement
![output1-01](https://user-images.githubusercontent.com/39416241/44006889-bf104680-9e59-11e8-922c-2317609b6ca6.png)

I chose religious words like: ['god', 'jesus', 'muslim', 'christ', 'holy', 'allah' ], I didn't add 'father', because I cannot define the context when they say father. The first year in 1982, there was one person been executed, which makes the value so big in the graph, I might drop the years of those numbers under 10 execution, maybe that will be more precise? 
![output2-01](https://user-images.githubusercontent.com/39416241/44006900-f40b4c22-9e59-11e8-9199-2faaba67f723.png)

## Any changes in direction or topic?

## Problems/Questions
Do I need to drop the years of those numbers under 10 execution?

## Checklist
	
- [x] I have included my visuals
- [x] I have filled out the sections above
- [x] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by xeophin** [comment]
Hi, looks like an interesting project.I would assume that there are some expectation what one has to say as last words, so it would be interesting to see whether you can find some "recipe" – but also whether there are some outliers, people that don't follow these rules. Looking forward to what you do with it.

**Comment by sarahslo** [comment]
yes you have to have a universe that doesn't skew the numbers. what do you mean by ratio? a ratio is a relationship between numbers. so is it religious words to overall words? definitely keep this to at least 10 executions.

and frequency, i'm not sure what you mean here. number of times each person said the word? can you write the definitions of what you are measuring out in the labels for clarity?

**Comment by SiruiZhu** [comment]
# Pre-Final

## Project visuals/text
![output1-01](https://user-images.githubusercontent.com/39416241/44295000-836f1e80-a26e-11e8-8829-e08964742706.png)
![output2](https://user-images.githubusercontent.com/39416241/44295001-836f1e80-a26e-11e8-9a95-68d87402ca2b.png)
![output4-01](https://user-images.githubusercontent.com/39416241/44295002-8407b500-a26e-11e8-83cc-ccddbfcda7b3.png)

Graph with note: https://siruizhu.github.io/Death%20Row%20Information/

## Details

**Headline:** 
Goodbye World! Last Statement Analysis for Criminals

**Published website version:**
https://siruizhu.github.io/Death%20Row%20Information/

**Code repository:**
https://github.com/SiruiZhu/Data-Studio/tree/master/code/05-death%20statement

**Final data set(s):** 
[Texas Department of Criminal Justice](http://www.tdcj.state.tx.us/death_row/dr_executed_offenders.html)

## What did you find to be the most difficult part of this project?
How to answer my question: The trend  the religious word been used over the years. I struggled how to figure this out.

## Are you satisfied with what you produced? Is there anything you would like to change or improve?
Not very, don't think my analysis is meaningful, kind of just get more familiar with text analysis process, but actually there are more need to be done. I also think the visualization should be better. 

Overall, this is a simple project, not much to explore.


## Checklist

- [x] I have included my visuals
- [x] I have posted my project to my project website
- [] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by sarahslo** [comment]
even tho this hasn't turned out as you expected, let's get the scales right. the rate is what? the number of times criminals have spoken a word...more than 0.5? i don't understand what you are counting or why there is a decimal involved.

**Comment by SiruiZhu** [comment]
@sarahslo Hi Sarah, thank you very much for your comment. The rate is I use the  religious word divided by the whole word criminals used in that year. So it's a decimal. Oh, do you mean I need to convert to rate, like 0.5 should be 50%... right?

**Comment by SiruiZhu** [comment]
# Final

## Project visuals/text
![output1-01](https://user-images.githubusercontent.com/39416241/44551928-9ebaad80-a6f6-11e8-9052-ec271da247ec.png)
![output2](https://user-images.githubusercontent.com/39416241/44551929-9ebaad80-a6f6-11e8-86c2-1c10bca74b6e.png)
![output4-01](https://user-images.githubusercontent.com/39416241/44551931-9ebaad80-a6f6-11e8-8c79-1931c2f5418e.png)


## Details

**Headline:** 
Goodbye World! Last Statement Analysis for Criminals

**Published website version:**
https://siruizhu.github.io/Death%20Row%20Information/

**Code repository:**
https://github.com/SiruiZhu/Data-Studio/tree/master/code/05-death%20statement

**Final data set(s):** 
[Texas Department of Criminal Justice](http://www.tdcj.state.tx.us/death_row/dr_executed_offenders.html)

## What did you find to be the most difficult part of this project?
How to answer my question: The trend  the religious word been used over the years. I struggled how to figure this out.

## Are you satisfied with what you produced? Is there anything you would like to change or improve?
Not very, don't think my analysis is meaningful, kind of just get more familiar with text analysis process, but actually there are more need to be done. I also think the visualization should be better. 

Overall, this is a simple project, not much to explore.


## Checklist

- [x] I have included my visuals
- [x] I have posted my project to my project website
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository
